static/bidder-info/ownadx.yaml

```diff
@@ -1,4 +1,4 @@
-endpoint: "https://pbs.prebid-ownadx.com/bidder/bid/{{.AccountID}}/{{.ZoneID}}?token={{.SourceId}}"
+endpoint: "https://pbs.prebid-ownadx.com/bidder/bid/{{.SeatID}}/{{.SspID}}?token={{.TokenID}}"
```

Comment:
Is it possible to pass `SeatID SspID` as query param? 

Consider that usage of partial dynamic urls is being discouraged and prebid team in future will be working to avoid usage of dynamic subdomain in endpoint url.
Major concerns with such usage are,

- security concerns
   The security aspect is alleviated by using a fixed top level domain. Due to the potential harm to hosts, we are strict in this requirement. We are working towards fixing the few adapters currently in violation.

- connection performance
   The connection performance advice is for your benefit. Client specific subdomains prevent Prebid Server from reusing connections across your clients which results in more connections needed for your adapter. The issue gets worse the more successful you are in attracting new clients. Hosts may choose to disable your adapter due to this behavior.

Reply:
Describe in issue https://github.com/prebid/prebid-server/issues/3774.
We added changes accordingly.
This is our standard endpoint for all client, if we change it its will empact on all other clients also.